backend/LeetWars.Builder/Services/XmlTestResultParserService.cs

```diff
@@ -0,0 +1,164 @@
+﻿using LeetWars.Builder.Interfaces;
+using LeetWars.Builder.Models;
+using System.Globalization;
+using System.Xml;
+
+namespace LeetWars.Builder.Services
+{
+    public class XmlTestResultParserService : IXmlTestResultParserService
+    {
+        public TestsOutput ParseCSharpTestResult(string xmlTestResult)
+        {
+            XmlDocument xmlDoc = new();
```

Comment:
why do not use "var" everywhere in this file for class definition?

Reply:
Fixed declarations list XmlDocument xmlDoc = new();